base/shell/explorer/traydeskbtn.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * PROJECT:     ReactOS Explorer
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Show Desktop tray button header file
+ * COPYRIGHT:   Copyright 2006-2007 Thomas Weidenmueller <w3seek@reactos.org>
+ *              Copyright 2018-2022 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ *              Copyright 2023 Ethan Rodensky <splitwirez@gmail.com>
+ */
+
+#pragma once
+
+// This window class name is CONFIRMED on Win10 by WinHier.
+static const WCHAR szTrayShowDesktopButton[] = L"TrayShowDesktopButtonWClass";
+#define TSDB_CLICK (WM_USER + 100)
+
+// The 'Show Desktop' button at the end of the taskbar
+class CTrayShowDesktopButton :
+    public CWindowImpl<CTrayShowDesktopButton, CWindow, CControlWinTraits>
+{
+    LONG m_nClickedTime;
+    HTHEME m_hTheme;
+    HTHEME m_hFallbackTheme;
+    MARGINS m_ContentMargins;
+    SIZE m_inset;
+    HICON m_icon;
+    SIZE m_szIcon;
+    BOOL m_highContrastMode;
+    BOOL m_drawWithDedicatedBackground;
+    BOOL m_bHovering;
+    HWND m_hWndTaskbar;
+
+public:
+    BOOL m_bPressed;
+    BOOL IsHorizontal;
```

Comment:
you prefer to keep `IsHorizontal`, or use `m_IsHorizontal` as the other members seems to be named? (I don't mind any version)

Reply:
To put what Hermes said into different words, be consistent with your naming convention. Either use the `m_` prefix on all of the members or none of them.